.travis.yml

```diff
@@ -8,8 +8,9 @@ env:
   - RUBY_GC_HEAP_GROWTH_MAX_SLOTS=300000
   - RUBY_GC_HEAP_INIT_SLOTS=600000
   - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
+  - RAILS_ENV=test
   matrix:
-  - TEST_SUITE=vmdb
+  - TEST_SUITE=vmdb PARALLEL='^spec/(controllers|helpers|initializers|javascripts|lib|mailers|models|presenters|requests|routing|services|task_helpers|tools|views)'
```

Comment:
I hate this but if this works, maybe we can clean up this hardcoded list of directories by organizing the test files in our tree based on the test suites.